File: AppGenerateFiles/dbo/Model/Catalogo_Municipio.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Catalogo_Municipio : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? id_municipio { get; set; }
       public string? nombre { get; set; }
       public int? id_departamento { get; set; }
       [ManyToOne(TableName = "Catalogo_Departamento", KeyColumn = "id_departamento", ForeignKeyColumn = "id_departamento")]
       public Catalogo_Departamento? Catalogo_Departamento { get; set; }
       [OneToMany(TableName = "Catalogo_Clientes", KeyColumn = "id_municipio", ForeignKeyColumn = "id_municipio")]
       public List<Catalogo_Clientes>? Catalogo_Clientes { get; set; }
       [OneToMany(TableName = "Catalogo_Inversores", KeyColumn = "id_municipio", ForeignKeyColumn = "id_municipio")]
       public List<Catalogo_Inversores>? Catalogo_Inversores { get; set; }
       [OneToMany(TableName = "Condicion_Laboral_Cliente", KeyColumn = "id_municipio", ForeignKeyColumn = "id_municipio")]
       public List<Condicion_Laboral_Cliente>? Condicion_Laboral_Cliente { get; set; }
   }
}

File: AppGenerateFiles/dbo/FrontModel/Catalogo_Municipio.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Catalogo_Departamento }  from './Catalogo_Departamento.js'
import { Catalogo_Clientes }  from './Catalogo_Clientes.js'
import { Catalogo_Inversores }  from './Catalogo_Inversores.js'
import { Condicion_Laboral_Cliente }  from './Condicion_Laboral_Cliente.js'
class Catalogo_Municipio extends EntityClass {
   constructor(props) {
       super(props, 'EntityDbo');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ id_municipio;
   /**@type {String}*/ nombre;
   /**@type {Catalogo_Departamento} ManyToOne*/ Catalogo_Departamento;
   /**@type {Array<Catalogo_Clientes>} OneToMany*/ Catalogo_Clientes;
   /**@type {Array<Catalogo_Inversores>} OneToMany*/ Catalogo_Inversores;
   /**@type {Array<Condicion_Laboral_Cliente>} OneToMany*/ Condicion_Laboral_Cliente;
}
export { Catalogo_Municipio }
